pegasus/sites.v3/code.org/public/css/learn-carousel.css

```diff
@@ -200,16 +175,36 @@
     width: 185px;
   }
 
-  .multi-item-carousel .carousel-inner .active.left {
-    left: -0%;
+  #banner-slideshow {
+    height: 80px;
   }
-  .multi-item-carousel .carousel-inner .active.right {
-    left: 0%;
+
+  @media all and (transform-3d), (-webkit-transform-3d) {
+    .carousel-inner > .item.next,
+    .carousel-inner > .item.active.right {
+      left: 0;
+      -webkit-transform: translate3d(50%, 0, 0);
```

Comment:
Ugh, right. So no animation in non-EN and IE9- but it's no worse than it is now. Just advances by 1, no slide animation. I'll keep tracking to get full support hopefully by next week. But at least getting this fix in for Thursday deadline.